lib/ansible/modules/cloud/amazon/route53.py

```diff
@@ -574,6 +574,13 @@ def main():
         else:
             wanted_rset.add_value(v)
 
+    need_to_sort_records = (type_in == 'CAA')
+
+    # Sort records for wanted_rset if necessary (keep original list)
+    unsorted_records = wanted_rset.resource_records
+    if need_to_sort_records:
+        wanted_rset.resource_records = sorted(unsorted_records)
```

Comment:
I think this could be more concise as 

```
if type_in == 'CAA':
    wanted_rset.resource_records.sort()
```

That way we'd just be using the in-place sort rather than creating new vars.

Reply:
But `unsorted_records` would no longer be the original copy, since it will be sorted as well.